postmortems/001 - Postmortem for v0.0.19 Deployment Failure.md

```diff
@@ -0,0 +1,54 @@
+# 000 - Postmortem for v0.0.19 Deployment Failure
```

Comment:
We can include links to PRs inline as links. I didn't do it because I wanted to keep these as short to write as possible so they aren't a large burden in the parts of the postmortem we care the least about (the timeline and other descriptions are for getting context, imo, not proving your abilities as a detective, and I'm more interested in the remedy section on how to avoid the issue in the future).